pkg/component/gardener/dashboard/config/config.go

```diff
@@ -41,3 +46,37 @@ type UnreachableSeeds struct {
 type ContentSecurityPolicy struct {
 	ConnectSources []string `yaml:"connectSrc"`
 }
+
+// Terminal is the configuration for the terminals.
+type Terminal struct {
+	Container                  TerminalContainer                   `yaml:"container"`
+	ContainerImageDescriptions []TerminalContainerImageDescription `yaml:"containerImageDescriptions"`
+	GardenTerminalHost         TerminalGardenHost                  `yaml:"gardenTerminalHost"`
+	Garden                     TerminalGarden                      `yaml:"garden"`
+}
```

Comment:
You suggest changing `container.image` to `containerImage`. However, that would also require changing the dashboard, as this struct reflects the dashboard config.

The main reason for using container.image was to future-proof the code in case we needed to add more properties related to the terminal container later, grouping them under one field to avoid breaking changes.

However, after years of experience, we never actually needed to add any properties, which wasn't entirely clear when we first developed the feature. In hindsight, for simplicity's sake, we should have used `containerImage` from the start

Reply:
Thanks for sharing.